### ISSUE TITLE
[A11y] MinimumTextContrastGuideline should exclude disabled component

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#94428]
As per https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/visual-audio-contrast-contrast.html

Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

MinimumTextContrastGuideline should ignored components that have isEnabled = false
